README.md

```diff
@@ -120,6 +120,7 @@ M1 Macbooks require some extra depencies in order to install Haystack.
 brew install postgresql
 brew install cmake
 brew install rust
+brew install sentencepiece
```

Comment:
I am bit surprised that we didn't notice this a s a problem earlier. Anyway, it's better to have this command here than a failing installation for new users! 👍

Reply:
I would have expected it to install automatically as a dependency of [sentence-transformers](https://github.com/UKPLab/sentence-transformers/blob/master/requirements.txt#L10) via pip install. At least that was the case for me. Maybe `pip install --upgrade pip` can make a difference? @bogdankostic do you remember whether you needed `brew install sentencepiece`?

Reply:
If I recall correctly, I did not need `brew install sentencepiece` either.

Reply:
I confirm that  the brew package is not needed, I must have been in a weird state when the error occurred the first time, reverting this change 👍